### PR TITLE
feat(calendar): option to always enable current date

### LINKED ISSUE #490: [Enhancement] Make current date in weekend selectable, when weekends are disabled
**Is your enhancement request related to a problem? Please describe.**
Eventhough weekend dates should generally not be selectable in our Transfer Flow, when transferring to an account where inter-bank clearing is not needed (eg. own accounts), it must always be possible to select current date as transfer date (even in weekends).

**Describe the solution you'd like**
An additional configuration property for the calendar component, eg. "alwaysEnableCurrentDate = true"

### PR DESCRIPTION
**Pull request by lydemann**
Closes #490
Enable selection of current date in calendar regardless of disabled date (e.g. disableWeekends=true, disabledDates=[...])

### CODE REVIEW

File: src/app/showcase/calendar-showcase/calendar-showcase.component.ts [hunk 1]
```diff
@@ -43,6 +44,12 @@ export class CalendarShowcaseComponent {
       defaultValue: 'false',
       inputValues: ['true | false'],
     },
+    {
+      name: 'alwaysEnableCurrentDate',
```
Comment (jakobe) [inline src/app/showcase/calendar-showcase/calendar-showcase.component.ts]:
```suggestion
      name: 'enableToday ',
```

File: src/app/showcase/calendar-showcase/calendar-showcase.component.ts [hunk 2]
```diff
@@ -43,6 +44,12 @@ export class CalendarShowcaseComponent {
...
+      description: '(Optional) Allows selection of current date regardless of disabled weekends.',
```
Comment (jakobe) [inline src/app/showcase/calendar-showcase/calendar-showcase.component.ts]:
```suggestion
      description: '(Optional) Allows selection of current date regardless of whether it is disabled (e.g. disableWeekends=true, disabledDates=[...]).',
```

File: src/kirby/components/calendar/calendar.component.ts [hunk 3]
```diff
@@ -42,6 +42,7 @@ export class CalendarComponent implements OnInit, AfterViewInit, OnChanges {
   @Input() disabledDates: Date[];
   @Input() minDate: Date;
   @Input() maxDate: Date;
+  @Input() alwaysEnableCurrentDate = false;
```
Comment (jakobe) [inline src/kirby/components/calendar/calendar.component.ts]:
```suggestion
  @Input() enableToday = false;
```

File: src/app/showcase/calendar-showcase/calendar-showcase.component.ts [hunk 4]
```diff
@@ -11,6 +11,7 @@ declare var require: any;
 })
 export class CalendarShowcaseComponent {
   disableWeekends = false;
+  alwaysEnableCurrentDate = false;
```
[the remark this answers is not in the document]
Comment (lydemann) [reply src/app/showcase/calendar-showcase/calendar-showcase.component.ts]: alwaysEnableToday is a more meaningful name as we talked about as enableToday = false sounds like today is disabled.

**Pull request merged**